app/assets/stylesheets/application.css.scss

```diff
@@ -40,6 +40,27 @@ legend {
   };
 }
 
+.reminder-container {
+  text-align: center;
```

Comment:
Properties should be ordered background-color, border, border-radius, margin, text-align, width